.gitlab-ci.yml

```diff
@@ -7,10 +7,16 @@ variables:
   BUILD_DEMO_REGISTRY: 172597598159.dkr.ecr.us-east-1.amazonaws.com/otel-collector-contrib
   BUILD_SANDBOX_REGISTRY: 601427279990.dkr.ecr.us-east-1.amazonaws.com/otel-collector-contrib
   DOCKERFILE: ci/Dockerfile
+  # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
+  # and check the job creating the image to make sure you have the right SHA prefix
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
+  # Make sure to update test-infra-definitions version in go.mod as well
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: b02a714d7de7
```

Comment:
Is this a pinned version of test infra definitions that will require updating in the future?

Reply:
Yeah it will require some update when you want to get update from `test-infra-definitions`. It updates the runner image that is used to execute the e2e tests.